packages/wallet/src/components/Menu.tsx

```diff
@@ -122,7 +122,7 @@ export const Menu = () => {
                         <Disclosure as="div" key={name} className="space-y-1">
                           {({ open }) => (
                             <>
-                              <Disclosure.Button className="flex w-full items-center justify-between text-xl text-brand-green-4">
+                              <Disclosure.Button className="flex w-full items-center justify-between text-xl text-brand-green-4 outline-none">
```

Comment:
If the outline issue was related to https://github.com/tailwindlabs/tailwindcss/pull/10385, we can remove `outline-none` now since this got reverted (https://github.com/tailwindlabs/tailwindcss/pull/10604).

Reply:
I think it is not related, menu items sometimes got an outline of blue color when clicking I think 2 times in a row on it

Reply:
![image](https://user-images.githubusercontent.com/117268143/220276225-6d4a79df-39ea-43e2-ab05-002fe3212d3f.png)